iptables/iptables.go

```diff
@@ -423,10 +424,18 @@ func existsRaw(table Table, chain string, rule ...string) bool {
 	return strings.Contains(string(existingRules), ruleString)
 }
 
-func filterOutput(output []byte, args ...string) []byte {
-	// ignore iptables' message about xtables lock
-	if strings.Contains(string(output), xLockWaitMsg) {
+// Maximum duration that an iptables operation can take
+// before flagging a warning.
+const opWarnTimeMsec = 2000
```

Comment:
I would move here the time.Millisecond moltiplication

Reply:
and also put `2 * time.Second`

Reply:
You're right, that would be cleaner.